### DISCUSSION TITLE
Extensions needed

### DISCUSSION THREAD

**Discussion by matteotestoni** [MahoCommerce/maho#19] [General]
Hi all,

Extensions needed for a modern ecommerce:
- brevo integration
- meta catalog integration
- google merchant center catalog integration
- stripe payment
- paypal payment
- customer attribute (fiscal code, if is private or not, etc)
- amazon ebay integration (m2epro)
- automatic related products
- recaptcha or turnstile for spam
- smtp email (ashroeder)
- one page checkout or a new checkout for maho
- improved layer navigation

**Comment by fballiano** [comment]
Hi @matteotestoni, I think we're covered about recaptcha/turnstile, paypal should work fine already but I agree about the rest

**Reply by matteotestoni** [reply]
i think the starting point maybe, can be the old extensions for magento 1, many are open source and in github
- https://github.com/facebookarchive/facebook-for-magento
- https://github.com/aschroder/Magento-SMTP-Pro-Email-Extension
- https://github.com/stripe-archive/stripe-magento1-releases
- https://github.com/sendinblue/magento1-plugin   
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
